integration/release_upgrade_linux_test.go

```diff
@@ -188,6 +189,75 @@ func shouldRecoverAllThePodsAfterUpgrade(t *testing.T, criRuntimeService cri.Run
 	}
 }
 
+func execToExistingContainer(t *testing.T, criRuntimeService cri.RuntimeService, criImageService cri.ImageManagerService) upgradeVerifyCaseFunc {
+	var busyboxImage = images.Get(images.BusyBox)
+
+	t.Logf("Pulling image %q", busyboxImage)
+	_, err := criImageService.PullImage(&criruntime.ImageSpec{Image: busyboxImage}, nil, nil)
+	require.NoError(t, err)
+
+	t.Log("Create sandbox")
+	sbConfig := PodSandboxConfig("sandbox", "running")
+	sbConfig.LogDirectory = t.TempDir()
+	sb, err := criRuntimeService.RunPodSandbox(sbConfig, "")
+	require.NoError(t, err)
+
+	t.Logf("Create a running container")
+	containerConfig := ContainerConfig("running", busyboxImage, WithCommand("sh", "-c", "while true; do date; sleep 1; done"))
+	containerConfig.LogPath = "running#0.log"
+	cntr, err := criRuntimeService.CreateContainer(sb, containerConfig, sbConfig)
+	require.NoError(t, err)
+	require.NoError(t, criRuntimeService.StartContainer(cntr))
+
+	// NOTE: Wait for containerd to flush data into log
+	time.Sleep(2 * time.Second)
+
+	return func(t *testing.T, criRuntimeService cri.RuntimeService) {
+		pods, err := criRuntimeService.ListPodSandbox(nil)
+		require.NoError(t, err)
+		require.Len(t, pods, 1)
+
+		cntrs, err := criRuntimeService.ListContainers(&criruntime.ContainerFilter{
+			PodSandboxId: pods[0].Id,
+		})
+		require.NoError(t, err)
+		require.Equal(t, 1, len(cntrs))
+		assert.Equal(t, criruntime.ContainerState_CONTAINER_RUNNING, cntrs[0].State)
+
+		func() {
```

Comment:
Why is this a separate func?

Reply:
Just want to highlight this one and I was thinking I should add a function to check log change. 
I can remove the func here.